spell-check.R

```diff
@@ -14,7 +14,10 @@ dict_file <- arguments[1]
 arguments <- arguments[-1]
 # if there are arguments, check those files, otherwise check all markdown & rmd files
 if (length(arguments) > 0 && arguments[1] != "") {
-  files <- arguments[grepl(file_pattern, arguments)]
+  files_glob <- strsplit(arguments, " ") |>
+    unlist() |>
+    Sys.glob()
```

Comment:
Just in case globs are overlapping at all
```suggestion
    Sys.glob() |>
    unique()
```